emergency_app/forms.py

```diff
@@ -0,0 +1,242 @@
+# Taken from this video:
+# https://www.youtube.com/watch?v=wVnQkKf-gHo
+from django import forms
+# from emergency_app.models.identity import Identity
+from emergency_app.models.contact import Contact
+from emergency_app.models.emergency import Emergency
+from emergency_app.models.nation import Nation
+# from emergency_app.models.state import State
+from common.util import sanitization
+
+# When required field=false, clean() would normalize empty value of CharField into empty string
+# However, null value was needed to follow the sample data provided. Therefore, this function is declared.
+# alternatively, to_python function for each field could use this function.
+# Source: https://docs.djangoproject.com/en/2.2/ref/forms/fields/
+def empty_string_handler(field):
+    # if field either empty string or empty/null value
+    if not field:
+        return None
+    else:
+        return field
+
+# Form for validating the request update of user's emergency contact
+class UpdateEmergencyContactForm(forms.ModelForm):
+    # Provide an example of the schema of the model
+    # pidm was needed to search all contacts related to related user.
+    pidm = forms.IntegerField()
+    surrogate_id = forms.IntegerField(required=False)
+    priority = forms.CharField(max_length=4)
+    relt_code = forms.CharField(max_length=4, required=False)
+    last_name = forms.CharField(max_length=240)
```

Comment:
First, last, and middle names each have a 60 character limit.  Here are the limits on address and phone records:

> SPRADDR_STREET_LINE1              VARCHAR2(75 CHAR) 
SPRADDR_STREET_LINE2              VARCHAR2(75 CHAR) 
SPRADDR_STREET_LINE3              VARCHAR2(75 CHAR) 
SPRADDR_CITY             NOT NULL VARCHAR2(50 CHAR) 
SPRADDR_STAT_CODE                 VARCHAR2(3 CHAR)  
SPRADDR_ZIP                       VARCHAR2(30 CHAR) 
SPRADDR_CNTY_CODE                 VARCHAR2(5 CHAR)  
SPRADDR_NATN_CODE                 VARCHAR2(5 CHAR)  
SPRADDR_PHONE_AREA                VARCHAR2(6 CHAR)  
SPRADDR_PHONE_NUMBER              VARCHAR2(12 CHAR) 
SPRADDR_PHONE_EXT                 VARCHAR2(10 CHAR)